tests/test_schema_utils.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from tests.schema_utils import (SchemasDirectoryNotFound, get_schema_file_path,
+                                get_schemas_base_path)
+
+
+def test_get_schemas_base_path_returns_base_path() -> None:
+    expected = f'/usr/local/src/libero-schemas'
+    base_path = get_schemas_base_path()
+    assert base_path == expected
```

Comment:
I'm wasn't sure what the best way to test this was so I'm happy to take suggestions for improvements.

Reply:
Yes, the hardcoded path seems brittle.

Do you care at all where the files are placed? I'd test that `get_schema_file_path` returns a path that can be `open()`ed and you can read some well-formed XML from it. More than a unit test this would be an integration test of the functions that have been built on top of `poetry` and its stack for installing libraries.

Reply:
I'm going to update this test and see if I can make it less brittle.

Reply:
updated.